Lombiq.Tests.UI/MonkeyTesting/MonkeyTestingOptions.cs

```diff
@@ -0,0 +1,125 @@
+        /// The default value is <see langword="true"/>.
+        /// </summary>
+        public bool RunBrowserLogAssertion { get; set; } = true;
+
+        /// <summary>
+        /// <para>Gets the gremlins' species.</para>
+        /// <para>By default contains:</para>
+        /// <list type="number">
+        /// <item><description><c>"gremlins.species.clicker()"</c></description></item>
+        /// <item><description><c>"gremlins.species.toucher()"</c></description></item>
+        /// <item><description><c>"gremlins.species.formFiller()"</c></description></item>
+        /// <item><description><c>"gremlins.species.scroller()"</c></description></item>
+        /// <item><description><c>"gremlins.species.typer()"</c></description></item>
+        /// </list>
+        /// </summary>
+        public List<string> GremlinsSpecies { get; } = new()
+        {
+            "gremlins.species.clicker()",
+            "gremlins.species.toucher()",
+            "gremlins.species.formFiller()",
+            "gremlins.species.scroller()",
+            "gremlins.species.typer()",
+        };
+
+        /// <summary>
+        /// <para>Gets the gremlins' mogwais.</para>
+        /// <para>By default contains:</para>
+        /// <list type="number">
+        /// <item><description><c>"gremlins.mogwais.alert()"</c></description></item>
+        /// <item><description><c>"gremlins.mogwais.gizmo()"</c></description></item>
+        /// </list>
+        /// </summary>
+        public List<string> GremlinsMogwais { get; } = new()
+        {
+            "gremlins.mogwais.alert()",
+            "gremlins.mogwais.gizmo()",
+        };
+
+        /// <summary>
+        /// Gets or sets the gremlins attack delay.
+        /// The default value is 10 milliseconds.
+        /// </summary>
+        public TimeSpan GremlinsAttackDelay { get; set; } = TimeSpan.FromMilliseconds(10);
+
+        /// <summary>
+        /// <para>Gets the URL cleaners.</para>
+        /// <para>By default contains:</para>
+        /// <list type="number">
+        /// <item><description><see cref="RemovesFragmentMonkeyTestingUrlCleaner"/> instance.</description></item>
+        /// <item><description><see cref="RemovesBaseUrlMonkeyTestingUrlCleaner"/> instance.</description></item>
+        /// <item><description><see cref="RemovesQueryParameterMonkeyTestingUrlCleaner"/> instance with <c>"admin"</c> argument.</description></item>
+        /// <item><description><see cref="RemovesQueryParameterMonkeyTestingUrlCleaner"/> instance with <c>"returnUrl"</c> argument.</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public List<IMonkeyTestingUrlCleaner> UrlCleaners { get; } = new()
+        {
+            new RemovesFragmentMonkeyTestingUrlCleaner(),
+            new RemovesBaseUrlMonkeyTestingUrlCleaner(),
+            new RemovesQueryParameterMonkeyTestingUrlCleaner("admin"),
+            new RemovesQueryParameterMonkeyTestingUrlCleaner("returnUrl"),
```

Comment:
Sorry, a comment in code? Or you don't understand why they are needed?

You wrote in requirements that "admin" parameter should be disregarded. I decided that "returnUrl", which occurs on Login page, should also be disregarded, as it doesn't change the page logic very much, and to avoid extra monkey cycles on Login page with different sources.

Reply:
A comment in code, yes. I understand the reason for these and I remember I asked for the admin one but for someone else looking at this piece of code it's not obvious.

Reply:
Added comments.